vmas/scenarios/transport.py

```diff
@@ -216,6 +246,23 @@ def reward(self, agent: Agent):
                 self.rew[package.on_goal] += 1.0 * self.package_on_goal_reward_factor
                 
 
+            # penalty (negative rew) for agent-agent collisions
+            for a in self.world.agents:
+                a.agent_collision_rew[:] = 0
+
+            for i, a in enumerate(self.world.agents):
+                for j, b in enumerate(self.world.agents):
+                    if i <= j:
+                        continue
+                    if self.world.collides(a, b):
+                        distance = self.world.get_distance(a, b)
```

Comment:
See previous comment concerining agents distance for collision calculation

Reply:
Also perimeter to perimeter (see above)